Calculator with simple ui.py

```diff
@@ -33,16 +86,16 @@ def divide(x, y):
         num2 = float(input("Enter second number: "))
 
         if choice == '1':
-            print(num1, "+", num2, "=", add(num1, num2))
+            print(calculator.add(num1, num2))
 
         elif choice == '2':
-            print(num1, "-", num2, "=", subtract(num1, num2))
+            print(calculator.subtract(num1, num2))
```

Comment:
Use a IDE if possible while coding.
Also, Use a code formatter if possible.
If possible, enable code formatter when saving the file.
You can also format the code by ```black```.
to install just type in cmd ```pip install black```
then format the file using cmd by typing in cmd ```black file_name.py```
and worry not, there are many ways to format a file. This clicks with me, that's it.